app/src/main/res/layout/activity_account.xml

```diff
@@ -84,11 +84,13 @@
 
           <Spinner
             android:id="@+id/currency_spinner"
+            style="@style/Base.Widget.AppCompat.Spinner.Underlined"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_marginEnd="@dimen/activity_horizontal_margin"
             android:layout_marginStart="@dimen/activity_horizontal_margin"
-            android:layout_marginTop="@dimen/grid_1">
+            android:layout_marginTop="@dimen/grid_1"
+            android:dropDownVerticalOffset="@dimen/grid_11">
```

Comment:
I don't think you'll need this because the drop down list is drawn over the spinner.

Reply:
Sounds good I'll remove it!